CONTRIBUTING.md

```diff
@@ -10,9 +10,9 @@
 * If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/rails/rails/issues/new). Be sure to include a **title and clear description**, as much relevant information as possible, and a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
 
 * If possible, use the relevant bug report templates to create the issue. Simply copy the content of the appropriate template into a .rb file, make the necessary changes to demonstrate the issue, and **paste the content into the issue description**:
-  * [**Active Record** (models, database) issues](https://github.com/rails/rails/blob/master/guides/bug_report_templates/active_record_master.rb)
-  * [**Action Pack** (controllers, routing) issues](https://github.com/rails/rails/blob/master/guides/bug_report_templates/action_controller_master.rb)
-  * [**Generic template** for other issues](https://github.com/rails/rails/blob/master/guides/bug_report_templates/generic_master.rb)
+  * [**Active Record** (models, database) issues](https://github.com/rails/rails/blob/main/guides/bug_report_templates/active_record_master.rb)
```

Comment:
Would you consider to rename `action_controller_master.rb` to `action_controller_main.rb` ? Here these file names based on the master branch. It also applies to other *_master.rb files.

Reply:
Hi @yahonda, I wasn't sure if that change should be included in this specific PR. I'd be happy to do that if it does.

Reply:
Looks like https://github.com/rails/rails/commit/077c66d5d6ef3a6f1cc54e4a431bfa5ea6831b97 already renamed the bug_report_templates file names.

Reply:
Thank you, I just merged those changes into this branch.

Reply:
Out of curiosity, was https://github.com/rails/rails/commit/077c66d5d6ef3a6f1cc54e4a431bfa5ea6831b97 a separate pull request that was opened previously before this one? I tried to find it, but I don't see it, or was it committed directly to the `main` branch absent a PR? As a first time contributor to the Rails docs, I'd like to know the process a bit more here. I had reviewed current PRs before opening this almost 5 days ago, and had not seen work in progress similar to it, so trying to understand how most of this was covered by that commit?

Reply:
There was no pull request opened for this change. As this is an administrative change that required coordinated changes to the doc server I decide to make the changes myself so we don't get in a broken state.